sql/2021/pwa/lighthouse_pwa_score.sql

```diff
@@ -0,0 +1,19 @@
+#standardSQL
+# Percentiles of lighthouse pwa score
+# This metric comes from Lighthouse only and is only available in mobile in HTTP Archive dataset
+SELECT
+  '2021_07_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
```

Comment:
I ran this query on `httparchive.sample_data.lighthouse_mobile_10k` and obtained the following results:

Row | Date            | Percentile | Score |
1      | 2021_07_01 | 10              | 0.25 |  
2      | 2021_07_01 | 25             | 0.33 |  
3      | 2021_07_01 | 50             | 0.42 |  
4      | 2021_07_01 | 75             | 0.5 |  
5      | 2021_07_01 | 90             | 0.5

Taking a look at the last row, does it mean that 90% of the reports have, **at least** a pwa score of 0.5?
Isn't that a very high value?

Reply:
Yup. Consistent with what I got before: https://www.tunetheweb.com/blog/what-do-lighthouse-scores-look-like-across-the-web/

My own blog gets a score of 0.5 despite not having a service worker not a manifest. There’s plenty of audits in this category that non-PWA websites can pass (e.g. uses HTTPS, loads fast enough, sets viewport… etc.).

Reply:
Isn't it the other way around? If the 90th percentile is 0.5, that means 10% of reports are at least 0.5. In other words, 90% of reports are less than 0.5.

Reply:
Sorry yes you’re right, misread @demianrenzulli ’s comment. 90% of pages have a score of 0.5 or less.

Btw I always thought that the percentile was inclusive (i.e. “90% of pages have a score of 0.5 or less.” rather than “90% of pages have a score of less than 0.5.”)? Not that that makes that much of a difference.

Reply:
Thanks for claryifing, folks.

As done in previous editions of the Web Almanac, I think that it would be interesting to present a conclusion that could be more or less actionable and informative, besides the raw data.

In this particular case, for example, would it be fair to assume that the most meaningful conclusion is:  "Only 10% of the reports have a PWA score higher or equals than 0.5?"

The other conclusion doesn't sound very useful: “90% of pages have a score of 0.5 or less.”. One could also also say something like "100% of pages have a score of less or equals than 1" and that would be right.

Leaving this discussion open so we can discuss in more detail when the time comes.

Reply:
You don’t have to use every query in the chapter. Some just won’t have any interesting insights and to be honest I think this is one of those that’s not very interesting. We didn’t use it in last two years despite running it. Should we drop it completely? Maybe. But on other have it’s written so no harm running. Who knows maybe this year it’ll surprise us.

I think the individual audits are more interesting and we did use them last year.